sklearn/cluster/tests/test_optics.py

```diff
@@ -190,226 +189,65 @@ def test_reach_dists():
     # Expected values, matches 'RD' results from:
     # http://chemometria.us.edu.pl/download/optics.py
```

Comment:
Have you made sure the results indeed match with this implementation?

Reply:
@lesteve I actually had not. Somehow I missed this comment. But I did now, and the results are not the same. Not even close actually. But I also checked if the values I'm replacing match the output of that code, and they don't match them. So at some point in time, either somebody forgot to check against the code provided in that URL, or that URL has changed the code and it doesn't match what it used to be. @espg any idea?

Reply:
I'll check for the iPython notebook where I generated the values; I think I remember having to sort the output by the ordering indexes to get them to match...